### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by ncdc**
Signed-off-by: Andy Goldstein <goldsteina@vmware.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,29 @@
...
-Our goal is to ultimately add this upgrade logic to Cluster API itself, but given that v1alpha2 doesn't easily lend itself to
-handling upgrades of control plane machines, we decided to build a temporarily tool that can fill that gap. Once Cluster API
-supports full lifecycle management of control planes, we plan to sunset this tool.
+**BACK UP** your control plane before you attempt an upgrade. This may include:
+
+- `Cluster` resource
+- `Machine` resources
+- `KubeadmConfig` resources
+- `Secret` resources containing the certificates and kubeconfig for your cluster
+- Contents of the `kube-system` namespace in the workload cluster (and any other namespaces/resources you may want)
```
Comment (detiber) [inline README.md]: This should likely also call out either taking an etcd snapshot or doing a full archive using Velero of the workload cluster being upgraded.

Comment (ncdc) [reply README.md]: Will do. Thanks!

Comment (ncdc) [reply README.md]: Added a to-squash commit - ptal!

Comment (detiber) [reply README.md]: the changes lgtm

Comment (ncdc) [reply README.md]: Squashed

**Pull request merged**